### ISSUE TITLE
Compatibility with RimWorld 1.1

### ISSUE THREAD

**Issue by SaucyPigeon** [SaucyPigeon/RW_ColonistBarHiding#16]

**Comment by SaucyPigeon** [comment]
Exception preventing UI from being loaded (not present in RimWorld version 1.0)

```
Root level exception in OnGUI(): System.MissingMethodException: string Verse.Translator.Translate(string)
  at ColonistBarHiding.Patches.ColonistBarColonistDrawer.ColonistBarColonistDrawer_HandleClicks.Postfix (UnityEngine.Rect rect, Verse.Pawn colonist) [0x00001] in <cb3977ca1ac24bba9c9572ee4e204b8d>:0 
  at (wrapper dynamic-method) RimWorld.ColonistBarColonistDrawer.HandleClicks_Patch2(object,UnityEngine.Rect,Verse.Pawn,int,bool&)
  at ColonistBarHiding.Patches.ColonistBar.ColonistBar_ColonistBarOnGUI.HandleOnGUI (System.Collections.Generic.List`1[T] colonistsToHighlight) [0x00068] in <cb3977ca1ac24bba9c9572ee4e204b8d>:0 
  at ColonistBarHiding.Patches.ColonistBar.ColonistBar_ColonistBarOnGUI.ColonistBarOnGUI (System.Collections.Generic.List`1[T] colonistsToHighlight) [0x00020] in <cb3977ca1ac24bba9c9572ee4e204b8d>:0 
  at ColonistBarHiding.Patches.ColonistBar.ColonistBar_ColonistBarOnGUI.Prefix (System.Collections.Generic.List`1[T] ___colonistsToHighlight) [0x00001] in <cb3977ca1ac24bba9c9572ee4e204b8d>:0 
  at (wrapper dynamic-method) RimWorld.ColonistBar.ColonistBarOnGUI_Patch1(object)
  at RimWorld.MapInterface.MapInterfaceOnGUI_BeforeMainTabs () [0x0003c] in <94d7dc7b4af948af8726d0765cd83ec8>:0 
  at RimWorld.UIRoot_Play.UIRootOnGUI () [0x0001f] in <94d7dc7b4af948af8726d0765cd83ec8>:0 
  at Verse.Root.OnGUI () [0x0003d] in <94d7dc7b4af948af8726d0765cd83ec8>:0 
Verse.Log:Error(String, Boolean)
Verse.Root:OnGUI()
```

**Comment by SaucyPigeon** [comment]
Fixed with version 1.1.5. RimWorld 1.1 uses different dlls and a different .NET version to 1.0.
